server/game/cards/09.4-TCoH/IkomaKiyono.js

```diff
@@ -0,0 +1,22 @@
+const DrawCard = require('../../drawcard.js');
+const AbilityDsl = require('../../abilitydsl');
+
+class IkomaKiyono extends DrawCard {
+    setupCardAbilities() {
+        this.wouldInterrupt({
+            title: 'Ready for Glory Count',
+            when: {
+                onGloryCount: (event, context) => { // eslint-disable-line no-unused-vars
+                    return context.player && context.player.opponent && context.player.honor > context.player.opponent.honor;
+                }
+            },
+            effect: 'ready for the incoming glory count.',
```

Comment:
not sure I like 'incoming' - isn't the default fine here?

Reply:
if 'player uses Ikoma Kiyono to ready Ikoma Kiyono is what you want?